castervoice/rules/ccr/rust_rules/rust.py

```diff
@@ -21,9 +21,9 @@ class Rust(MergeRule):
             R(Text("else {}") + Key("left/5:3")),
         #
         SymbolSpecs.SWITCH:
-            R(Text("match")),
+            R(Text("match ")),
         SymbolSpecs.CASE:
-            R(Text(" => ") + Key("left")),
+            R(Text(" => ")),
```

Comment:
After inserting a => It is always followed by space and text begins there. I can imagine why the key left was in there, unless it was a copy paste from a different language grammar.